addon/components/detail-instructors-list.js

```diff
@@ -13,6 +14,14 @@ export default Component.extend({
   sortGroupsBy: null,
   sortedInstructors: sort('instructors', 'sortInstructorsBy'),
   sortedInstructorGroups: sort('instructorGroups', 'sortGroupsBy'),
+  instructorGroupMembers: computed('instructorGroups.[]', async function() {
+    const instructorGroups = await this.instructorGroups;
+    const groupsOfInstructors = await all(instructorGroups.mapBy('users'));
+    return groupsOfInstructors.reduce((array, set) => {
```

Comment:
I believe this can now be done with [Array.flat()](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/flat). Not 100% certain though.

Reply:
`flat()` works, but as it turns out those CPs are not even necessary, they were kruft left over from a previous interation of this component rework. removed.